src/css/components/_volume.scss

```diff
@@ -19,7 +19,6 @@
 
 .video-js .vjs-volume-control {
   cursor: pointer;
-  margin-right: 1em;
```

Comment:
The volume control component can be added to the control bar separately and this would affects that.
Probably needs to reset the margin-right just in `.video-js .vjs-volume-panel` below (line 38/37)